### PR TITLE
French fix

### PR DESCRIPTION
**Pull request by checksum0**
Further improvement to French translations, especially in regards to uniformity with Spanish translations. Also added clickable link.

### CODE REVIEW

File: src/culture/fr.js [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
 		"vanitywallet": "Porte-monnaie de courtoisie",
-		"splitwallet": "Porte-monnaie divisé",
+		"splitwallet": "Porte-monnaie partagé",
+		"splitlabelthreshold": "Nombre minimale de parts requises pour combiner",
+		"splitlabelshares": "Nombre de parts totale",
+		"splitview": "Générer les parts",
 		"detailwallet": "Détails du porte-monnaie",
 
 		// footer html
 		"footerlabeldonations": "Don: 1LTzRmha779gYTEcDFoXgSP8DsUeZMYk6H",
```
Comment (zquestz) [inline src/culture/fr.js]: This should be `1AqBBnvDqhaXvzaLnZxSwxBVQ7JnEMBihS`

File: src/culture/fr.js [hunk 2]
```diff
@@ -68,15 +71,16 @@
...
+		"singletip3": "<b>Pour ajouter</b> des fonds à votre porte-monnaie, envoyez des Bitcoin Cash à votre adresse de réception.",
+		"singletip4": "<b>Vous pouvez vérifier le contenu de votre porte-monnaie</b> en consultant https://blockchair.com/bitcoin-cash/blocks o https://cash.coin.dance/ et en y tapant votre adresse de réception Bitcoin Cash.",
```
Comment (zquestz) [inline src/culture/fr.js]: The URL's here should be links =)

**Comment by zquestz** [comment]
A few comments, but overall looks good. =)

File: src/culture/fr.js (hunk 1 above)
Comment (checksum0) [reply src/culture/fr.js]: Corrected!

File: src/culture/fr.js (hunk 2 above)
Comment (checksum0) [reply src/culture/fr.js]: Added!

**Pull request merged**